src/tools/spin/spinquic.cpp

```diff
@@ -533,9 +533,162 @@ struct SetParamHelper {
     }
 };
 
+void SpinQuicRandomizeSettings(QUIC_SETTINGS& Settings, uint16_t ThreadID)
```

Comment:
I am guessing this function is added as a means to stress test (randomly assign settings to quic), Maybe this is why some of the stress tests are crashing?

Reply:
Yes, this function is about randomly assigning settings. But the "crash" was from a watchdog timer which fires if the test takes too long. So, I wonder if it's caused by some timer getting set to too big which then causes the delay. I haven't had time to investigate yet though. @ProjectsByJackHe this might be a good place for you to do your first investigation.

Reply:
I'm not sure which setting is causing the watchdog timeout. I'm going to disable most of these to try to narrow down the scope. We can follow up with other PRs to enable more.